### PR TITLE
modulesync 2.2.0

### PR DESCRIPTION
**Pull request by bastelfreak**
modulesync 2.2.0

### CODE REVIEW

**Comment by bastelfreak** [comment]
hey @dhollinger, can I assign this to you?

File: .travis.yml [hunk 1]
```diff
@@ -18,13 +18,31 @@ matrix:
...
   - rvm: 2.5.1
+    bundler_args: --without system_tests development release
+    env: PUPPET_VERSION="~> 6.0" CHECK=rubocop
+  - rvm: 2.4.4
     bundler_args: --without system_tests development release
     env: PUPPET_VERSION="~> 5.0" CHECK=build DEPLOY_TO_FORGE=yes
+  - rvm: 2.5.1
+    bundler_args: --without development release
+    dist: trusty
+    env: PUPPET_INSTALL_TYPE=agent BEAKER_IS_PE=no BEAKER_PUPPET_COLLECTION=puppet5 BEAKER_debug=true BEAKER_setfile=centos6-64{hypervisor=docker} CHECK=beaker
```
Comment (dhollinger) [inline .travis.yml]: CentOS 6.x is not currently supported by this module. It requires Rsyslog 7.x+ and CentOS 6's default is Rsyslog 5.x

File: .travis.yml [hunk 2]
```diff
@@ -37,6 +55,30 @@ matrix:
...
+  - rvm: 2.5.1
+    bundler_args: --without development release
+    dist: trusty
+    env: PUPPET_INSTALL_TYPE=agent BEAKER_IS_PE=no BEAKER_PUPPET_COLLECTION=puppet6-nightly BEAKER_debug=true BEAKER_setfile=centos7-64{hypervisor=docker} CHECK=beaker
+    services: docker
+    sudo: required
+  - rvm: 2.5.1
+    bundler_args: --without development release
+    dist: trusty
+    env: PUPPET_INSTALL_TYPE=agent BEAKER_IS_PE=no BEAKER_PUPPET_COLLECTION=puppet5 BEAKER_debug=true BEAKER_setfile=debian8-64{hypervisor=docker} CHECK=beaker
```
Comment (dhollinger) [inline .travis.yml]: Debian 8 doesn't appear to work with this module either. This will need to be investigated before we can support it.

**Comment by bastelfreak** [comment]
ah. I think they were added to the `.sync.yml` but the `.travis.yml` wasn't updated. I dropped them again from `.sync.yml`

**Pull request merged**